frontend/src/Settings/MediaManagement/MediaManagement.js

```diff
@@ -130,6 +130,16 @@ class MediaManagement extends Component {
                         isAdvanced={true}
                         size={sizes.MEDIUM}
                       >
+                        <FormLabel>Use Reflinks instead of Copy</FormLabel>
```

Comment:
May want to have this only available as an option on compatible platforms.

Reply:
I agree... But I have no clue how to do that :D

Reply:
The code for macOS isn't all that complicated though (it is for Windows ReFS).

Maybe I should just add some extra documentation in the warning part of the label?

Reply:
I have added the warning in 4de62be90.